### DISCUSSION TITLE
Ender 3 v2 bed dimensions

### DISCUSSION THREAD

**Discussion by QuayleMan89** [mriscoc/Ender3V2S1#329] [General]
[Ender3V2-422-BLT-20220816.bin] Version  build plate dimension is set to 230x230 when build plate only 220x220 causing gridding on the X when trimming.  
Solution  Go to the Advance tab, then physical settings and adjust both the x and y size and x and y max position setting.

**Comment by cvail55** [comment]
Not true, my bed is 236mm, bone stock. We just set it to 220 to clear bed clips. I have a Microswiss DD and a stock hotend and both can reach 230mm on x before anything gets close.

**Comment by JEBurroughs** [comment]
No issues either.
https://www.youtube.com/watch?v=ygol-UwtfkU

**Comment by mriscoc** [comment]
The dimensions of the Ender3v2 glass bed is 235x235 mm, the dimensions on the firmware are calibrated for a totally stock Ender 3V2 printer without any modification.
If you made any hardware modification please read: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
![image](https://user-images.githubusercontent.com/2745567/187094810-20d79a18-3d5e-45ea-ae75-91399aa23d17.png)
